src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,96 @@
 package juja.sqlcmd;
 
+import java.sql.*;
```

Comment:
Pay attention https://google.github.io/styleguide/javaguide.html 3.3.1

Reply:
I fixed it. Thanks for link.